api/envoy/config/accesslog/v2/file.proto

```diff
@@ -17,5 +18,8 @@ message FileAccessLog {
   // Access log format. Envoy supports :ref:`custom access log formats
   // <config_access_log_format>` as well as a :ref:`default format
   // <config_access_log_default_format>`.
-  string format = 2;
+  oneof access_log_format {
+    string format = 2;
+    google.protobuf.Struct json_format = 3;
```

Comment:
Can you add a comment describing how this is interpreted? E.g. can we use the same custom access log formats in leaf strings, limitations on depth of nesting, etc.

Reply:
Yup, this will be coming soon with the docs updates.
- [ ] protobuf comment